.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
This file is auto-generated by Husky.

Reply:
Is this file supposed to be committed? My understanding was that husky automatically setup pre-commit hooks for users in a `postinstall` script.